group-coordinator/src/test/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroupTest.java

```diff
@@ -675,25 +675,25 @@ public void testValidateOffsetCommit() {
 
         // Simulate a call from the admin client without member id and member epoch.
         // This should pass only if the group is empty.
-        group.validateOffsetCommit("", "", -1);
+        group.validateOffsetCommit("", "", -1, false);
```

Comment:
Do we want to have a test where this argument is true?

Reply:
`isTransactional` has actually no impact on the validation but I agree that we should check to ensure this. I have parameterized it to check both.